safe-global/web-core
# Fix: show upgrade notification only to owners

## What it solves
The upgrade notification is only actionable if you're the owner.
Noticed this when opening the demo safe.

## How to test
1)
* Open a 1.1.1 Safe that doesn't belong to you (or simply disconnect the wallet)
* The upgrade notification should NOT pop up

2)
* Open a 1.1.1 Safe as an owner
* Observe the notification
